src/core/stream.h

```diff
@@ -152,6 +152,8 @@ typedef union QUIC_STREAM_FLAGS {
 
         BOOLEAN InStreamTable           : 1;    // The stream is currently in the connection's table.
         BOOLEAN DelayIdFcUpdate         : 1;    // Delay stream ID FC updates to StreamClose.
+
+        BOOLEAN ShutdownReliableSend    : 1;    // Indicates that we should shutdown the send path once we sent/ACK'd ReliableOffsetSend bytes.
```

Comment:
Put this right after `LocalCloseReset` and call it (for now) `LocalCloseResetReliable`.

Reply:
Done